packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3455,6 +3455,19 @@ class RenderSemanticsGestureHandler extends RenderProxyBox {
     markNeedsSemanticsUpdate();
   }
 
+  /// If non-null, the set of actions to override.
+  ///
+  /// Set by the [RawGestureDetectorState] if the switch control in IOS is
+  /// enabled to create custom action for scrolling.
```

Comment:
... if switch control on iOS is enabled ...

Reply:
This doc also needs a little more context describing what this is usually used for (see the docs for validActions for an example)